ssl/statem/statem_dtls.c

```diff
@@ -116,6 +116,7 @@ int dtls1_do_write(SSL *s, int type)
     size_t curr_mtu;
     int retry = 1;
     size_t len, frag_off, mac_size, blocksize, used_len;
+    unsigned char msg_hdr_backup[DTLS1_HM_HEADER_LENGTH];
```

Comment:
Theses changes in this file should better be in a separated commit, as it does not seem to be related to RFC support.
I remember reading your comments about this change, may be in a separated Issue, or PR ?

Reply:
There is already a PR with exactly these changes: #18111 
I expect these changes to disappear once that is merged.